README.md

```diff
@@ -1,3 +1,101 @@
-# STARC
+![Starc Logo](./img/starc.png)
 
-## Story Architect is a writer's best friend
+  
+
+# Welcome to Story Architect
+
+  
+
+Hello and welcome to **Starc's** GitHub page. Story Architect is a project created by the authors of an open source screenwriting tool [**Kit Scenarist**](https://github.com/dimkanovikov/KITScenarist) and is currently under development with its current release being alpha. It's an advanced text editor for script writers working professionally in film, theatre, radio or writing for comic books.
```

Comment:
I suggest mentioning what concrete features are not been implemented yet to not make readers think that they are already available. Like plays, mindmaps, images, etc.